utils/create_validate_pr_quickstarts.js

```diff
@@ -416,6 +417,11 @@ const main = async () => {
 
   const hasFailed = await createValidateUpdateQuickstarts(files);
 
+  newrelic.recordCustomEvent('validate_or_update_quickstarts', {
```

Comment:
i think we can improve the name of the event.

Pascal case looks to be the standard for event names / types users can query from.